### PR TITLE
Update kappurumedia.json Adding discord verif

### PR DESCRIPTION
**Pull request by CyrusCore**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

kappurumedia.is-a.dev

### CODE REVIEW

File: domains/kappurumedia.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "email": "abramsatria7@gmail.com"
   },
   "record": {
+    "TXT": "dh=ce5c5f60ae66a7f508b44432fe2d9a0b51038fa8",
```
Comment (DEV-DIBSTER) [inline domains/kappurumedia.json:9]: Cannot have TXT and CNAME records together.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/kappurumedia.json (hunk 1 above)
Comment (CyrusCore) [reply domains/kappurumedia.json:9]: how to add discord verif then

Comment (CyrusCore) [reply domains/kappurumedia.json:9]: add another json file?

Comment (DEV-DIBSTER) [reply domains/kappurumedia.json:9]: > add another json file?

Yes, it would be _discord.kappurumedia.json

**Pull request closed without merging**